packages/react-devtools-shared/src/devtools/views/DevTools.js

```diff
@@ -127,6 +127,35 @@ export default function DevTools({
     [enabledInspectedElementContextMenu, viewAttributeSourceFunction],
   );
 
+  const devToolsRef = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    const ownerWindow = devToolsRef.current.ownerDocument.defaultView;
+
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (event.ctrlKey || event.metaKey) {
+        switch (event.key) {
+          case '1':
+            setTab(tabs[0].id);
+            event.preventDefault();
+            event.stopPropagation();
+            break;
+          case '2':
+            setTab(tabs[1].id);
+            event.preventDefault();
+            event.stopPropagation();
+            break;
+        }
+      }
+    };
+
+    if (showTabBar) ownerWindow.addEventListener('keydown', handleKeyDown);
```

Comment:
nit: Let's move this check to the start of the effect? No point to define `handleKeyDown` (or return a cleanup function) if we're not going to use it. :smile: